scripts/postinstall.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+docker stop mama-ng-scheduler
+docker rm mama-ng-scheduler
+
+docker stop postgresql
+docker rm postgresql
+
+# Exit on errors from here.
+set -e
+
+docker run -d --name="postgresql" \
+    -v `pwd`/data:/data:rw \
```

Comment:
mounts a local volume for the data, postgres will write its DBs here ensuring it lives on during redeploys.